anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py

```diff
@@ -46,57 +52,71 @@ def get_stats_accidents_by_car_type_with_national_data(
         data_by_segment = AccidentCountByCarTypeWidget.percentage_accidents_by_car_type(
             involved_by_vehicle_type_data
         )
-        national_data = AccidentCountByCarTypeWidget.percentage_accidents_by_car_type_national_data_cache(
-            start_time, end_time
+        national_data = (
+            AccidentCountByCarTypeWidget.percentage_accidents_by_car_type_national_data_cache(
+                start_time, end_time
+            )
         )
 
-        for k, v in national_data.items():  # pylint: disable=W0612
+        for car_type in data_by_segment:
             out.append(
                 {
-                    "car_type": k,
-                    "percentage_segment": data_by_segment[k],
-                    "percentage_country": national_data[k],
+                    "label_key": car_type,
+                    "series": [
+                        {"label_key": "percentage_segment", "value": data_by_segment[car_type]},
+                        {"label_key": "percentage_country", "value": national_data[car_type]},
+                    ],
                 }
             )
         return out
 
     @staticmethod
     def percentage_accidents_by_car_type(involved_by_vehicle_type_data):
-        driver_types = defaultdict(float)
+        vehicle_type_dict = defaultdict(float)
         total_count = 0
         for item in involved_by_vehicle_type_data:
-            vehicle_type, count = item["involve_vehicle_type"], int(item["count"])
+            vehicle_type, count = item["vehicle_type"], int(item["count"])
             total_count += count
             if vehicle_type in VehicleCategory.CAR.get_codes():
-                driver_types[VehicleCategory.CAR.value] += count
+                vehicle_type_dict[VehicleCategory.CAR.value] += count
             elif vehicle_type in VehicleCategory.LARGE.get_codes():
-                driver_types[VehicleCategory.LARGE.value] += count
+                vehicle_type_dict[VehicleCategory.LARGE.value] += count
             elif vehicle_type in VehicleCategory.MOTORCYCLE.get_codes():
-                driver_types[VehicleCategory.MOTORCYCLE.value] += count
+                vehicle_type_dict[VehicleCategory.MOTORCYCLE.value] += count
             elif vehicle_type in VehicleCategory.BICYCLE_AND_SMALL_MOTOR.get_codes():
-                driver_types[VehicleCategory.BICYCLE_AND_SMALL_MOTOR.value] += count
+                vehicle_type_dict[VehicleCategory.BICYCLE_AND_SMALL_MOTOR.value] += count
             else:
-                driver_types[VehicleCategory.OTHER.value] += count
+                vehicle_type_dict[VehicleCategory.OTHER.value] += count
+
+        output = {}
+        # Order the vehicle_type_dict dict by value which is the count of the vehicle_type
+        vehicle_type_dict_ordered = dict(

```

Comment:
you can perform the ordering on the original dict, no need for additional variable